calculator.js

```diff
@@ -0,0 +1,32 @@
+var value = 0;
```

Comment:
Great start with Bonus 1!

What would your code look like if `value` was a property of your calculator object instead of a global variable? Can you think of any reasons why that would be a better way of setting up your calculator?